src/Microsoft.ComponentDetection.Detectors/poetry/PoetryComponentDetector.cs

```diff
@@ -7,10 +7,12 @@
 using Microsoft.ComponentDetection.Contracts.Internal;
 using Microsoft.ComponentDetection.Contracts.TypedComponent;
 using Microsoft.ComponentDetection.Detectors.Poetry.Contracts;
-using Nett;
 
 namespace Microsoft.ComponentDetection.Detectors.Poetry
 {
+    using System.IO;
```

Comment:
shift these 2 using statements outside of namespace alogn with other using statements.